docs/csharp/language-reference/compiler-options/publicsign-compiler-option.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "-publicsign (C# Compiler Options)"
+ms.date: 05/15/2018
+f1_keywords: 
+  - "/publicsign"
+helpviewer_keywords: 
+  - "-publicsign compiler option [C#]"
+  - "publicsign compiler option [C#]"
+  - "/publicsign compiler option [C#]"
+---
+# -publicsign (C# Compiler Options)
+
+This option causes the compiler to apply a public key without actually signing the assembly. The **-publicsign** option also sets a bit in the assembly that tells the runtime that the file is actually signed.
```

Comment:
Ing. Maybe "This option causes the compiler to apply a public key without an assembly sign."?